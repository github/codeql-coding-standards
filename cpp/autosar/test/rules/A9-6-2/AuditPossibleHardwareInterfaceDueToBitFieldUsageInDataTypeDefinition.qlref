rules/A9-6-2/AuditPossibleHardwareInterfaceDueToBitFieldUsageInDataTypeDefinition.ql