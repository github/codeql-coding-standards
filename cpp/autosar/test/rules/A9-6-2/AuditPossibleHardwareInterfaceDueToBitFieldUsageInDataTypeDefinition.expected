| test.cpp:22:7:22:8 | a1 | Usage of bit-field in data type definition. |
| test.cpp:47:9:47:10 | a2 | Usage of bit-field in data type definition. |
| test.cpp:50:8:50:9 | c1 | Usage of bit-field in data type definition. |
| test.cpp:57:8:57:8 | c | Usage of bit-field in data type definition. |
